traveler-swift-core/TravelerKit/Traveler.swift

```diff
@@ -51,6 +51,17 @@ public class Traveler {
 
     func identify(_ identifier: String?, attributes: [String: Any?]?) {
         self.session.identity = identifier
+
+        guard let attributes = attributes, let travelerProfileId = session.identity else {
+            return
+        }
+        let requestOperation = AuthenticatedRemoteRequestOperation(path: .storeAttributes(attributes, travelerId: travelerProfileId), session: session)
+        let blockOperation = BlockOperation { [unowned requestOperation] in
+            Log("There's an error storing user attributes.", data: requestOperation.error, level: .error)
```

Comment:
You're not guarding for the error here. This means that this log will get executed every time. The fact that the error in the request operation is nil, doesn't stop the log operation from executing, the only difference is that when the call is successful, the log will not display data and when the call is unsuccessful the log will display data.